stainless_frontend/tests/pass/type_class.rs

```diff
@@ -18,7 +18,7 @@ trait Equals {
     // => this.equals(self, x)
     !self.equals(x)
   }
-
+  /*
```

Comment:
Commented out the laws to avoid measure checking. But in the last test case, it fails anyway because of measure inference.